src/styles/icons.css

```
@@ -1,120 +1,122 @@
 @font-face {
-    font-family: 'MusescoreIcon';
-    src: url('/fonts/MusescoreIcon.ttf') format('truetype');
-    font-weight: normal;
-    font-style: normal;
-    font-display: swap;
+  font-family: "MusescoreIcon";
```

Stylelint (17.14.0)

[error] Expected no quotes around "MusescoreIcon" (font-family-name-quotes)

(font-family-name-quotes)